### DISCUSSION TITLE
Not cleaning fully

### DISCUSSION THREAD

**Discussion by gpsnomad** [leon-cleaning-services/leon#162] [Q&A] [answered]
Hi. Am I doing something wrong? 
![Screenshot_20230507-150314](https://user-images.githubusercontent.com/476693/236682374-8624c4cf-39f6-4085-896d-40179788dbe6.png)

**Comment by svenjacobs** [comment, marked as answer]
Hello @gpsnomad,

you're not doing anything wrong :) `utm_*` parameters are removed because these are generic parameters occurring on many websites. However `leadId` and `mkt_tok` are special to `fastcompany.com` so we would have to write a sanitizer for this domain. I'm glad if you could help me here and create a pull request 😃

**Reply by gpsnomad** [reply]
Hello @svenjacobs 

Thanks for the reply. Has been a while since I coded anything! 😅

Any chance you could build an option that just strips all parameters? Ie parses the link and stops at the first question mark that it finds? That would be good enough for me, rather than a custom sanitizer for each domain. 

Also, just to note that in your Android app description on the Play Store, the link to GitHub is broken because it has an extra full stop at the end.
